### ISSUE TITLE
Training script

### ISSUE THREAD

**Issue by roholazandie** [google-deepmind/regress-lm#73]
Thanks for the excellent work! Will the training script be available soon?

**Comment by xingyousong** [comment]
Hi - the standard pretraining class is here: https://github.com/google-deepmind/regress-lm/blob/main/regress_lm/pytorch/pretraining.py

it can be used in any main file (with hyperparameter tuning over batch size, learning rate, etc.).
